src/tcgwars/logic/impl/gen3/UnseenForces.groovy

```diff
@@ -1207,7 +1207,7 @@ public enum UnseenForces implements LogicCardInfo {
             damage 20
             afterDamage {
               if (opp.hand) {
-                def list = opp.hand.shuffledCopy().showToMe("Opponent's hand").filterByType(TRAINER)
+                def list = opp.hand.shuffledCopy().filterByType(TRAINER)
```

Comment:
While not doing a show, then select sounds good, the player should still be able to see the whole hand.

This could be replaced with a select that makes use of a filter, to confirm you're selecting a trainer or skip. If player picked a card, it is discarded.

Reply:
Good point, with my latest change I've made it so it shows the whole hand, only allowing selection of the trainer cards.